app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,2 @@
+module SessionsHelper
```

Comment:
why empty helper kept?

Reply:
@khadzhinov I think of using  the module SessionHelper for controller actions  log_in, log_out and other non CRUD sessions actions. They will all be in one place. I will include it in aplication controller.
One convenience I found, if I am correct, is that such helpers are automatically included in the Rails views.